README.md

```diff
@@ -34,6 +39,31 @@ It is possible to send args to the script under test by adding -- after the load
 ./bin/load-runner.js -s ./scripts/dummy.js -c 5 -n 100 -r 10 -o -- -testarg1 testval1
 ```
 
+# Environment variables
+
+There are some environment variables generated for the test script that can be useful. You can use Node.js `process.env` object to access them.
+
+* `LR_FLOW_NUMBER` - number generated each run, value depends on percentages specified by `-f` parameter, see [Flow numbers](#flow-numbers)  
+* `LR_TOTAL_RUNS` - total number of runs, it's value of `-n` parameter
+* `LR_RUN_NUMBER` - current test run number
+* `LR_RAND` - randomly generated value for each run, float between 0 and 1, based on random generation with `--seed` parameter
+
+Look at [scripts/dummy_env_vars.js](scripts/dummy_env_vars.js) for example of using this.
+
+
+# Flow numbers
+
+Flow number functionality can be used inside test script to determine which branch of script to be executed. 
+It's specified by `-f` parameter and can be accessed by `LR_FLOW_NUMBER` environment variable in the script.
+
+For example this executes 5 times flow 0, 3 times flow 1, 2 times flow 2 from total of 10 runs.
+
+```
+./bin/load-runner.js -s ./scripts/dummy_flow_numbers.js -n 10 -o -f 50 30 20 --seed 123 
+```
+
+Flow numbers are randomly shuffled, but it's possible to use `--seed` parameter for repeatable result. 
```

Comment:
@aidenkeating I think that's a valid point: we're only guaranteed repeatability when using the same seed *and* the same total number of runs. E.g., if we have 100 total runs, and see a spike at a particular point, caused by a clustering of a particular flow, that spike may not be there if we then changed the total runs to 101, because the pattern would change.

@wojta I don't understand most of the words in your reply, but I think I see the point you're trying to make: why does this have to be so complicated! :)

I think we might be trying to do too much, since any implementation here should be able to cater for *n* number of flows. We could maybe get what we want more easily by having the logic we need in the actual test script, rather than here in load-runner, since we know upfront that it's only going to be two flows. Each iteration of the test script could decide, based on its own *runNum*, what flow it should execute.

I'm happy to go with consensus, if we can reach any. I understand the desire to make something more generally useful here in load-runner, but doing the simplest thing is also good sometimes. :)